kernel/src/backtrace.rs

```diff
@@ -1,3 +1,5 @@
+use core::intrinsics::size_of;
```

Comment:
You can replace it with `use core::mem::size_of;`
to avoid `#![feature(core_intrinsics)]` 😂

Reply:
It somehow failed to appear when I am searching for `size_of`. Fixed.